wordpress-application/task-definition.tf

```diff
@@ -0,0 +1,54 @@
+data "aws_iam_role" "ecs_role" {
+  name = "ecsTaskExecutionRole"
+}
+
+resource "aws_ecs_task_definition" "ecs_task_definition" {
+  family             = "${var.project_name}-task-definition"
+  execution_role_arn = data.aws_iam_role.ecs_role.arn
+
+  container_definitions = <<TASK_DEFINITION
+  [
+    {
+      "name": "wordpress-container",
+      "image": "160372566358.dkr.ecr.ap-southeast-2.amazonaws.com/group3-wordpress:latest",
+      "cpu": 10,
+      "memory": 512,
+      "essential": true,
+
+      "secrets": [
+        { 
+          "name": "WORDPRESS_DB_HOST",
+          "valueFrom": "arn:aws:ssm:ap-southeast-2:160372566358:parameter/WORDPRESS_DB_HOST"
```

Comment:
make the account number a variable. you can get that from a data resource
same for region

`data "aws_caller_identity" "current" {}`
`data "aws_region" "current" {}`

`arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter/WORDPRESS_DB_HOST`

Reply:
do the same for other secrets